content/200-orm/200-prisma-client/500-deployment/301-edge/100-overview.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Overview'
+metaTitle: 'Overview (Deploy Prisma ORM at the Edge)'
+metaDescription: 'Learn how to deploy your Prisma-backed apps to edge functions like Cloudflare Workers or Vercel Edge Functions'
+tocDepth: 2
+---
+
+## Deploying edge functions with Prisma ORM
+
+You can deploy an application that uses Prisma ORM to the edge. Depending on which edge function provider and which database you use, there are different considerations and things to be aware of.
+
+Here is a brief overview of all the edge function providers that are currently supported by Prisma ORM:
```

Comment:
Wouldn't it make sense to show the overview of provider and driver here?
That would highlight that Neon, PlanetScale and Turso work on both, but that `pg` only works on Cloudflare.
It could also mention that there are not edge compatible drivers for MongoDB and SQL Server for example.

That you can use Accelerate on all of them could just be a note under the table, which then also can mention that any other database (MongoDB, MySQL, SQL Server custom PostgreSQL) is supported via Accelerate as well.

Reply:
fair enough, I had initiated a discussion about the table on Slack when I first created it: https://prisma-company.slack.com/archives/C04CED9G127/p1706188668281619

in the end, I went with this version because it seemed the most simple and clearly highlighted the value of Accelerate. I can experiment with other versions of this table though that include your suggestion 👍

Reply:
I agree, to highlight Accelerate this makes total sense - but I think the focus should be to focus what Prisma Client can do here. The table as it was yesterday felt more like something I would have expected in the Accelerate docs (maybe with the columns exchanged, so that Accelerate is listed first).